controllers/tab/catalogEntry/form/CatalogEntryCatalogMetadataForm.inc.php

```diff
@@ -121,6 +122,7 @@ function initData() {
 			'licenseURL' => $submission->getDefaultLicenseURL(),
 			'arePermissionsAttached' => !empty($copyrightHolder) || !empty($copyrightYear) || !empty($licenseURL),
 			'confirm' => ($this->_publishedMonograph && $this->_publishedMonograph->getDatePublished())?true:false,
+			'datePublished' => $submission->getDatePublished(),
```

Comment:
I believe it is not necessary to provide datePublished here, when it is provided in the fetch function -- the fetch function is/should be always called at the end, after the init function?

Reply:
:+1: